src/db/postgres/lrsql/postgres/sql/delete.sql

```diff
@@ -61,3 +61,30 @@ DELETE FROM credential_to_scope
 WHERE api_key = :api-key
 AND secret_key = :secret-key
 AND scope = :scope::scope_enum;
+
+----------------------begin components of delete-actor-----
+-- :name delete-actor-and-dependents!
+-- :command :execute
+-- :result :affected
+DELETE FROM statement_to_statement 
+WHERE ancestor_id IN (SELECT statement_id FROM statement_to_actor
+WHERE actor_ifi = :actor-ifi)
+OR descendant_id IN (SELECT statement_id FROM statement_to_actor
+WHERE actor_ifi = :actor-ifi);
+
+DELETE FROM statement_to_activity WHERE statement_id IN (SELECT statement_id FROM statement_to_actor
+WHERE actor_ifi = :actor-ifi);
```

Comment:
So some suggestions on formatting:

I would start the subqueries on a separate line, to make it easier to tell where the main delete op ends and where the subquery begins. I would also put linebreaks before keywords like `WHERE` and `IN` in order to shorten line lengths and make the query easier (for humans) to parse. So I'd reformat the command like so:
```sql
DELETE FROM statement_to_activity
WHERE statement_id
IN (
  SELECT statement_id
  FROM statement_to_actor
  WHERE actor_ifi = :actor-ifi
);
```

Reply:
amended as described